hu.json

```diff
@@ -280,15 +280,15 @@
 			"option-catalyst-desc": "Jelenleg {{tier}} Katalizátor licenc rendelkezésére áll. Köszönjük a támogatását!",
 			"option-catalyst-desc-no-license": "Jelenleg nincs Katalizátor licenc. A Katalizátor licenc egyszeri adomány az Obsidian fejlesztésének támogatására.",
 			"label-app": "Alkalmazás",
-			"label-add-own-language": "See how to add your language to Obsidian."
+			"label-add-own-language": "Nézze meg, hogy hogyan lehet hozzáadni a saját nyelvét az Obsidianhoz."
```

Comment:
Please don't translate the word "Obsidian".
It's a brand name and should not be translated.

Reply:
I didn't translate it. I just added affix to it